motor/core.py

```diff
@@ -104,6 +104,7 @@ class AgnosticClient(AgnosticBaseProperties):
     address                  = ReadOnlyProperty()
     arbiters                 = ReadOnlyProperty()
     close                    = DelegateMethod()
+    __hash__                 = DelegateMethod()
```

Comment:
This is needed to support putting MotorClients into a set/dict ala PYTHON-2466.

Reply:
👍🏽 ... Should there be a Motor ticket for adding this feature to motor? Or can we just add a changelog entry to Motor mentioning that this is now possible?

Reply:
I just mentioned it in the changelog.